src/win32-misc.h

```diff
@@ -40,4 +40,10 @@ int inet_pton(int af, const char *src, void *dst);
 
 #define geteuid() (0)
 
+#ifdef MAX_PATH
```

Comment:
I think this should move to `suricata-common.h` and check for `NAME_MAX` instead, and define it if undefined.